src/components/AgmExtension/ExtensionRequest.vue

```diff
@@ -0,0 +1,158 @@
+          <v-col
+            cols="12"
+            sm="3"
+          >
+            <strong>Is this the first AGM?</strong>
+          </v-col>
+          <v-col
+            cols="12"
+            sm="9"
+          >
+            <v-radio-group
+              v-model="data.isFirstAgm"
+              class="mt-0 pt-0"
+            >
+              <v-radio
+                label="Yes"
+                :value="true"
+              />
+              <v-radio
+                label="No - Specify AGM Year below"
+                :value="false"
+              />
+            </v-radio-group>
+          </v-col>
+        </v-row>
+
+        <v-row no-gutters>
+          <v-col
+            cols="12"
+            sm="3"
+          >
+            <strong>AGM Year</strong>
+          </v-col>
+          <v-col
+            cols="12"
+            sm="9"
+          >
+            <v-text-field
+              v-model="agmYear"
+              class="agm-year-textfield"
+              filled
+              label="AGM Year"
+            />
+          </v-col>
+        </v-row>
+      </div>
+
+      <v-divider class="mx-4" />
+
+      <div class="px-6 py-7">
+        <v-row no-gutters>
+          <v-col
+            cols="12"
+            sm="3"
+          >
+            <strong>Previous AGM date or a reference date</strong>
+          </v-col>
+          <v-col
+            cols="12"
+            sm="9"
+          >
+            <DatePicker />
+          </v-col>
+        </v-row>
+
+        <v-row no-gutters>
+          <v-col
+            cols="12"
+            sm="3"
+          >
+            <strong>Has an extension been requested for this AGM year already?</strong>
+          </v-col>
+          <v-col
+            cols="12"
+            sm="9"
+          >
+            (future)
+          </v-col>
+        </v-row>
+      </div>
+
+      <v-divider class="mx-4" />
+
+      <div class="px-6 py-7">
+        <v-row no-gutters>
+          <v-col
+            cols="12"
+            sm="3"
+          >
+            <strong>Intended date this AGM will be held</strong>
+          </v-col>
+          <v-col
+            cols="12"
+            sm="9"
+          >
+            <DatePicker />
+          </v-col>
+        </v-row>
+      </div>
+    </template>
+  </VcardTemplate>
+</template>
+
+<script lang="ts">
+import { Component, Emit, Prop, Vue } from 'vue-property-decorator'
+import { VcardTemplate } from '@/components/common'
+import { AgmExtEvalIF } from '@/interfaces'
+import { DatePicker } from '@bcrs-shared-components/date-picker'
+
+@Component({
+  components: {
+    DatePicker,
+    VcardTemplate
+  }
+})
+export default class ExtensionRequest extends Vue {
+  @Prop({ required: true }) readonly data!: AgmExtEvalIF
+
+  @Emit('data')
+  private emitData () {
+    return this.data
+  }
+
+  get agmYear (): string {
+    return this.data.agmYear
+  }
+
+  set agmYear (value: string) {
+    this.data.agmYear = value
+    this.emitData()
+  }
```

Comment:
... But actually, `agmYear` is both a getter (from data prop) and a setter, which emits/syncs the object back to the parent component.

In practise, the other approach is simpler (see `isFirstAgm`), but since this component will also contain logic and compute data, emitting/syncing will also be needed.